Interop/Indexer/src/main/kotlin/org/jetbrains/kotlin/native/interop/indexer/Indexer.kt

```diff
@@ -44,7 +44,7 @@ private interface ObjCContainerImpl {
     val properties: MutableList<ObjCProperty>
 }
 
-private class ObjCProtocolImpl(
+class ObjCProtocolImpl(
```

Comment:
I've done this for both ObjcProtocolImpl, and ObjCClassImpl, as otherwise I would not be able to define my own classes/protocols in J2ObjcParser. Is there a way of doing this without changing the existing code?